### PR TITLE
fix: use tagline in meta og:description tags on project pages

### PR DESCRIPTION
**Pull request by filipviz**
This PR previously added utils to remove specific HTML tags from descriptions. Now, it just uses the tagline.

### CODE REVIEW

File: src/lib/utils.ts [hunk 1]
```diff
@@ -21,3 +21,26 @@ export function cnTextHw(className: string | undefined) {
     className.includes('text-xs') && 'h-3 w-3',
   ]
 }
+
+/**
+ * Removes headers and HTML tags from a string for use in e.g. meta og:description tags.
+ * @param description - The description to sanitize.
+ * @returns The sanitized description.
+ */
+export function sanitizeDescriptionContent(description: string) {
```
Comment (wraeth-eth) [inline src/lib/utils.ts:30]: Looking good! Is it possible to write some tests for this to make sure it is robust?

File: src/pages/p/[projectId]/index.tsx [hunk 2]
```diff
@@ -17,13 +18,16 @@ export default function Page({
   metadata,
 }: InferGetStaticPropsType<typeof getStaticProps>) {
   const pid = projectId ? BigInt(projectId) : 0n
+  // Sanitize before slicing to prevent partial HTML tags at the end.
+  const sanitizedDescription = metadata?.description
```
Comment (wraeth-eth) [inline src/pages/p/[projectId]/index.tsx:22]: Can we move this to the SEO component so we get the functionality on all SEO descriptions?

**Review by wraeth-eth — CHANGES_REQUESTED** [review]
Awesome work, @filipviz ! Just two requests

File: src/pages/p/[projectId]/index.tsx (hunk 2 above)
Comment (filipviz) [reply src/pages/p/[projectId]/index.tsx]: Not needed, as project pages were the only page with HTML in their descriptions (others are hard-coded to typical strings).

File: src/lib/utils.ts (hunk 1 above)
Comment (filipviz) [reply src/lib/utils.ts]: Removed. Just using taglines instead.

**Pull request merged**